Release v1.1.2

dylanporyan opened the pull request:
## Updates in v1.1.2
The File Level Metadata reporting format was updated in August of 2025 to address feedback from adopters. The changes made in v1.1.2 focused on providing clarification on metadata fields within the file level metadata and data dictionary files, and providing examples for the use of `header_rows` and `column_or_row_name_position` fields. No changes were made to the templates directly, therefore, these changes are backwards compatible and parsable by the ESS-DIVE Fusion Database. 

Recommendations captured in #43, which were addressed in this PR.

* Provide clarity for `header_rows` field definition (File Level Metadata)
    * Revise the definition of the `header_rows` field to provide additional clarity to avoid misinterpretation
    * Revise the definition of the `header_rows` field to provide links to the example images
* Revise `column_or_row_name_position` field description to include examples (File Level Metadata)
    * Revise the definition of the `column_or_row_name_position` field to provide links to the example images
* Incorporate examples of using `header_rows` and `column_or_row_name_position` fields (File Level Metadata)
    * Provide examples of using the `header_rows` and `column_or_row_name_position` fields: [example 1](flmd_header_rows_example_1.png), [example 2](flmd_header_rows_example_2.png), [example 3](flmd_header_rows_example_3.png)
* Revise FLMD instructions file (File Level Metadata)
    * Provide clarity on the use of the `header_rows` and `column_or_row_name_position` fields and link to examples within the FLMD instructions
* Provide clarity for `missing_value_code` field (Data Dictionary)
    * Provide clarity on the `missing_value_code`, including that the field is to be used to specify a single missing value code
* Revise ReadMe file to describe changes of v1.1.2
    * Revised the ReadMe file to describe the changes in v1.1.2

Closes #43

dylanporyan committed:
New Examples, Addresses Issue #43

Adding images for example use of the header_rows and column_or_row_name_position fields

dylanporyan committed:
Revision to flmd quick guide, Addresses Issue #43

Revision of the descriptions for the header_rows and column_or_row_name_position fields. Inclusion of links to examples.

dylanporyan committed:
Revision to instructions, Addresses Issue #43

Addition of details on the use of the header_rows and column_or_row_name position fields under the "Notes" section

dylanporyan committed:
Revision to dd quick guide, Addresses Issue #43

Update to the missing_value_code description to clarify confusion

dylanporyan committed:
Update README.md to describe v1.1.2

dylanporyan committed:
Delete flmd_header_rows_example_1.png

dylanporyan committed:
Delete flmd_header_rows_example_2.png

dylanporyan committed:
Delete flmd_header_rows_example_3.png

dylanporyan committed:
Upload revised example images

Revised example images based on user suggestions. Uploading revised versions.

dylanporyan committed:
Revise flmd_quick_guide.md descriptions

Revise descriptions within the FLMD quick guide based on feedback.

dylanporyan committed:
Update flmd_instructions.md notes

Update notes field within FLMD instructions based on feedback

mburrus reviewed (changes requested):
@dylanporyan I have revised the release draft to be more concise and direct. Please revise the REAME to match this formatting.

dylanporyan committed:
Update README.md

Update README to revise description of updates made in v1.1.2 to make the description more succinct

dylanporyan commented:
@mburrus I've revised the README to match the revised description of v1.1.2 changes, as described in the release draft. Thank you for reviewing.

mburrus reviewed (approved):
Looks great.
